File: 3588/3588.cs
public class Solution {
    public int FindMaximumAreaOfATriangle(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(FindMaximumAreaOfATriangle(root.left), FindMaximumAreaOfATriangle(root.right));
    }
}

File: 3588/3588.go
func findMaximumAreaOfATriangle(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := findMaximumAreaOfATriangle(root.Left)
	r := findMaximumAreaOfATriangle(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}